tutorials/focus-mode/focus-mode.css

```
@@ -0,0 +1,12 @@
+/* 
+Copyright 2021 Google LLC
+Use of this source code is governed by a BSD-style
+license that can be found in the LICENSE file or at
+https://developers.google.com/open-source/licenses/bsd 
+*/
+
+body > .scaffold > :is(top-nav, navigation-rail, side-nav, footer),
+main > :not(:last-child),
+main > :last-child > navigation-tree {
+  display: none;
+}
```

Add some padding to the top and bottom of the article content. The whitespace makes it feel a little less cramped.

```suggestion
}

main > :last-child {
  margin-top: min(10vmax, 10rem);
  margin-bottom: min(10vmax, 10rem);
}
```